scripts/lxc-pstart

```diff
@@ -56,7 +56,20 @@ class SubpError(IOError):
             (pre + data.replace(cr, cr + pre) + cr).decode(errors='ignore'))
 
 
+class Container(object):
+    def __init__(self, ref):
+        if ':' in ref:
+            self.remote, _, self.name = ref.partition(":")
+        else:
+            self.remote = ""
+            self.name = ref
+
+    def __str__(self):
+        return self.remote + ":" + self.name if self.remote else self.name
+
+
 def lxc(args, ofmt=None, rcs=None, fmsg=None, data=None):
+    args = [str(a) for a in args]
```

Comment:
This is what converts input to 'lxc' to a string.
so: lxc("start", Container("foo")) 
does what is expected.

Reply:
Huh.  That's surprising to me.